platform/security/handler/security-handler-pki/src/main/java/org/codice/ddf/security/handler/pki/AbstractPKIHandler.java

```diff
@@ -103,7 +103,13 @@ public HandlerResult getNormalizedToken(
     } else {
       String errorMsg = "The certificate used to complete the request has been revoked.";
       LOGGER.info(errorMsg);
-      throw new ServletException(errorMsg);
+      try {
+        httpResponse.sendError(HttpServletResponse.SC_FORBIDDEN, "Your certificate is revoked.");
```

Comment:
❓  was there an agreement on the error message here?

Reply:
It is a  false positive because the `catch` block will catch the nullpointerexception. However, it might be cleaner to handle a `null` value explicitly. something like this:

```
if (httpResponse == null) {
        LOGGER.error("Error returning revoked certificate request because the HTTP response object is invalid.");
}
else {
   try {
        httpResponse.sendError(HttpServletResponse.SC_FORBIDDEN, "Your certificate is revoked.");
        httpResponse.flushBuffer();
      } catch (Exception e) {
        LOGGER.error("Error returning revoked certificate request.");
      }
}```